### PR TITLE
Add vue3 support

### PR DESCRIPTION
**Pull request by DSergiu**
* setup yarn and lerna monorepo
* move current setup to `packages/vue2`
* add `packages/vue3` deployed as `@hcaptcha/vue3`
* add `vue3` examples: traditional, vanilla script
* update ci/cd

TODO:
- [x] add vue3 tests
- [x] update READMEs

### CODE REVIEW

**Review by e271828- — DISMISSED** [review]
Nice! See minor comments below.

File: examples/traditional-vue3/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "traditional-vue3",
+  "version": "0.0.0",
```
Comment (e271828-) [inline examples/traditional-vue3/package.json:3]: Should this be changed?

File: packages/vue2/package.json [hunk 2]
```diff
@@ -0,0 +1,121 @@
+{
+  "name": "@hcaptcha/vue-hcaptcha",
+  "version": "0.3.0",
```
Comment (e271828-) [inline packages/vue2/package.json:3]: Bump this?

File: packages/vue3/package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@hcaptcha/vue3",
+  "version": "0.0.1",
```
Comment (e271828-) [inline packages/vue3/package.json:3]: Maybe start it at 1.0? This will never get auto-upgrades.

**Review by e271828- — APPROVED** [review]

**Comment by brdlyptrs** [comment]
Does this now deploy two separate npm packages?

File: packages/vue3/package.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@hcaptcha/vue3",
```
[the remark this answers is not in the document]
Comment (DSergiu) [reply packages/vue3/package.json:2]: Yes, we will deploy vue3 as a separate package. What name would you suggest? `@hcaptcha/vue3-hcaptcha` ? @brdlyptrs 

Regarding the other pr #40, it was requiring more work in setting up a mono repo, adjusting ci/cd pipeline, adding tests for vue3.

**Pull request merged**